src/routes/+page.svelte

```diff
@@ -34,6 +35,25 @@
     })),
   };
 
+  let params = $page.url.searchParams;
```

Comment:
does this need to be reactive? I'm not sure how query string changes are propagated...